packml_msgs/msg/ItemizedStats.msg

```diff
@@ -0,0 +1,6 @@
+# This message contains itemized statics for PackML statistics reporting
+# Each item containds a unique
+
+int16 id                          # unique identifier (user defined)
+int32 count                       # number of occurances
+std_msgs/Duration duration        # duration of in error state
```

Comment:
It's a left over from refactoring.  The idea behind the `ItemizedStats` is to be used to generate Pareto information.  The `duration` may or may not apply.  The idea of item statistics is not in the packml standard, but it's something we can generate because we have more flexibility in a C++/python program